conans/client/conan_api.py

```diff
@@ -456,6 +460,31 @@ def download(self, reference, remote_name=None, package=None, recipe=False):
         else:
             raise ConanException("Provide a valid full reference without wildcards.")
 
+    @api_method
+    def workspace_install(self, path, settings=None, options=None, env=None,
+                          remote_name=None, build=None, profile_name=None,
+                          update=False, cwd=None):
+        cwd = cwd or get_cwd()
+        abs_path = os.path.normpath(os.path.join(cwd, path))
+
+        workspace = Workspace(abs_path, self._cache)
+        graph_info = get_graph_info(profile_name, settings, options, env, cwd, None,
+                                    self._cache, self._user_io.out)
+        self._user_io.out.success("Using workspace file from %s" % workspace._base_folder)
```

Comment:
Instead of the `base_folder` we should inform here of the file actually used (and better not to use a private member).

Reply:
Removed, this doesn't make sense anymore that it will fail if the file is not found.